index.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Chocolate Pizza | Delicious</title>
+    <meta name="Learn how to make a Chocolate Pizza dessert with this easy to follow recipe, complete with ingredients list." content="">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/grid.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="container_16">
+      <header>
+        <section class="header grid_16">
+          <section class="grid_5">
+            <img src="images/logo.jpg" class="grid_1 alpha">
+            <h1 class="pageTitle">Delicious</h1>
+            <h2 class="pageTagline">The Best Food Blog on the Web.</h2>
+          </section>
+          <section class="contact grid_5 omega">
+            <img src="images/facebookIcon.jpg" alt="Facebook">
+            <img src="images/twitterIcon.jpg" alt="Twitter">
+            <img src="images/googleIcon.jpg" alt="Google+">
+            <img src="images/instagramIcon.jpg" alt="Instagram">
+            <img src="images/flickrIcon.jpg" alt="Flickr">
+            <img src="images/pinterestIcon.jpg" alt="Pinterest">
+            <img src="images/rssIcon.jpg" alt="RSS">
+            <img src="images/emailIcon.jpg" alt="Email">
+          </section>
+        </section>
+        <img src="images/lineBreak.jpg" class="lineBreak">
```

Comment:
The line break is better achieved through background image property. Apply it to an existing element, such as the article.